content/docs/introduction/faq.md

```diff
@@ -30,6 +30,17 @@ The Alertmanager cannot currently be made highly available, but this is a goal.
 
 Most Prometheus components are written in Go. Some are also written in Java and Ruby.
 
+### How stable are Prometheus features, storage formats, and APIs?
+
+Although Prometheus and many of its ecosystem components are already quite
+stable, we will still allow for occasional breaking changes until the
+Prometheus server reaches version 1.0.0. These breaking changes will be pointed
+out in release announcements for components that already have a proper release
+process (like the Prometheus server) or communicated clearly otherwise. After
+releasing version 1.0.0, we will switch to a true [semantic
```

Comment:
This true is confusing. Breaking APIs for versions <1.0.0 is still "true" semver. It's explicitly noted as that.